internal/backends/mcu/renderer.rs

```diff
@@ -13,6 +13,7 @@ use i_slint_core::{Color, ImageInner};
 
 use crate::fonts::FontMetrics;
 use crate::lengths::PhysicalPx;
+use crate::profiler::Timer;
```

Comment:
Either rename it ProfilerTimer or always use `profiler::Timer` because i find just `Timer` a bit ambiguous.

```suggestion
use crate::profiler;
```